app/controllers/invites_controller.rb

```diff
@@ -20,8 +20,8 @@ def accept
       set_website(@invite.website)
       # it will redirect to errors url because we set the website for the user
       after_login_redirect
-    else
-      # it will redirect to create website wizard because we don't sent it
+    elsif @invite.email.casecmp(current_user.email) == 1
+      # it will redirect to website wizard url because emails don't match
       after_login_redirect
```

Comment:
Why did you add this? We don't care if the email is different. We should redirect to `after_login_redirect` in call cases. In fact there is probably other way than that. So why the explicit check?